tensorflow.xml

```diff
@@ -2,7 +2,7 @@
 <manifest project="swift">
   <remote name="github" fetch="git://github.com" />
 
-  <default revision="master" />
+  <default revision="trunk" />
```

Comment:
This impacts a number of repositories bellow (i.e. anything that does not explicitly specify a revision) so I think that we should change this to `main` and update any of the revisions that explicitly are using `main` to that, and use the explicit name for the other ones.